test/e2e/e2e_test.go

```diff
@@ -47,9 +48,45 @@ var _ = ginkgo.BeforeSuite(func() {
 	deployerBaseDir = filepath.Clean(filepath.Join(basedir, "..", ".."))
 	binariesPath = filepath.Clean(filepath.Join(deployerBaseDir, "_out"))
 	fmt.Fprintf(ginkgo.GinkgoWriter, "using binaries at %q\n", binariesPath)
+	deploy()
```

Comment:
this means we do deploy after each suite, isnt't it? we should rather do it only in the overall test(s) - for now.

Reply:
I used ```BeforeAll()``` which is some wrapper to ```BeforeEach()``` that run only once, but I couldn't find any equivalent for that when I wanted to remove everything, IOW ```AfterAll()``` can't work the same. Do you have any other idea?

Reply:
I'd just do it on `BeforeEach`/`AfterEach` on the new test which checks the correctness of the deploy. If more tests will require this wrapping, we will re-evaluate.

Reply:
Deploy/Remove for evey new ```It()``` it's kinda heavy don't you think?

Reply:
Probably. So let's call it explicitely and manually just inside `ginkgo.Context("with cluster with all the deployed components", func() {`

Reply:
The only drawback here is that we won't be able to remove everything in case of an error in the test. ```defer()``` won't help either